app/views/admin/api_docs/base/_form.html.slim

```diff
@@ -9,7 +9,8 @@
     = form.input :system_name, input_html: { :disabled => true }
   = form.input :published, label: 'Publish?'
   = form.input :description, input_html: {rows: 3}
-  = form.input :service_id, as: :select, collection: @service_apis
+  - unless @api_docs_service.new_record? && @api_docs_service.service_id.present?
```

Comment:
Probably turning this into

`if @api_docs_service.persisted? || @api_docs_service.service_id.blank? `